cloud/organization/organization_token.go

```diff
@@ -158,35 +227,356 @@ func getOrganizationToken(id, name, message string, tokens []astrocore.ApiToken)
 				return astrocore.ApiToken{}, err
 			}
 		}
-	case name != "" && id != "":
-		return astrocore.ApiToken{}, errBothNameAndID
 	}
 	if token.Id == "" {
 		return astrocore.ApiToken{}, errOrganizationTokenNotFound
 	}
 	return token, nil
 }
 
-// get all organization tokens
-func getOrganizationTokens(client astrocore.CoreClient) ([]astrocore.ApiToken, error) {
+// List all organization Tokens
+func ListTokens(client astrocore.CoreClient, out io.Writer) error {
 	ctx, err := context.GetCurrentContext()
 	if err != nil {
-		return []astrocore.ApiToken{}, err
+		return err
+	}
+	organization := ctx.Organization
+
+	apiTokens, err := getOrganizationTokens(client)
+	if err != nil {
+		return errors.Wrap(err, astro.AstronomerConnectionErrMsg)
```

Comment:
```suggestion
		return fmt.Errorf("%s: %w", astro.AstronomerConnectionErrMsg, err)
```
nit: moving to package "errors"